<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Drone_State</class>
 <widget class="QWidget" name="Drone_State">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="General">
     <item>
      <layout class="QHBoxLayout" name="Connection">
       <item>
        <layout class="QVBoxLayout" name="Configuration">
         <item>
          <widget class="QLabel" name="GestionConnection_label">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Gestion des connexions</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="SeperatorConnection_line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="LineEditConfiguration">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Nom de la sauvegarde</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="save_textfield"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="SaveConfiguration_pushButton">
           <property name="text">
            <string>Sauvegarder les informations de connexion</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="DroneChoice_horizontalLayout">
           <item>
            <widget class="QLabel" name="DroneChoice_label">
             <property name="text">
              <string>Charger les informations</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="DroneChoice_comboBox"/>
           </item>
           <item>
            <widget class="QPushButton" name="refresh_pushButton">
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="myResource.qrc">
               <normaloff>:/new/myResource/img/refresh.png</normaloff>:/new/myResource/img/refresh.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="DeveloperMode">
           <item>
            <widget class="QCheckBox" name="DeveloperMode_checkBox">
             <property name="text">
              <string>Mode développeur (connexion requise)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="Prompt">
           <item>
            <widget class="QLabel" name="command_label">
             <property name="text">
              <string>Commande:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="prompt_lineEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="prompt_pushButton">
             <property name="text">
              <string>Envoyer</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextEdit" name="LogHistory_textfield">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="LogAction">
           <item>
            <widget class="QPushButton" name="Vider">
             <property name="text">
              <string>Vider le contenu</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Sauvegarder">
             <property name="text">
              <string>Sauvegarder</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="SeparatorCenter_line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="ConnectionAndGetInfo">
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="Connexion">
           <item>
            <widget class="QLabel" name="ConnexionTitle_label">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Connexion</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="LoadedFile">
             <item>
              <widget class="QLabel" name="loadedfile_label">
               <property name="text">
                <string>Fichier Chargé:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="loadedfilename_label">
               <property name="text">
                <string>Aucun</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="Hostname">
             <item>
              <widget class="QLabel" name="Hostname_label">
               <property name="text">
                <string>Serveur :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Hostname_textfield">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="Port">
             <item>
              <widget class="QLabel" name="Port_label">
               <property name="text">
                <string>Port :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Port_textfield">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line1">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="Auth">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Clé SSH (laisser vide si authentification par mot de passe) :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="LoadPath_button">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Parcourir</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="PrivateKey">
             <item>
              <widget class="QLineEdit" name="SSHKey_textfield">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="Username">
             <item>
              <widget class="QLabel" name="Username_label">
               <property name="text">
                <string>Nom d'utilisateur :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Username_textfield">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="Password">
             <item>
              <widget class="QLabel" name="Password_Label">
               <property name="text">
                <string>Password :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Password_textfield">
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="echoMode">
                <enum>QLineEdit::Password</enum>
               </property>
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="Interaction">
             <item>
              <widget class="QPushButton" name="buttonConnect">
               <property name="text">
                <string>Connexion</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonReset">
               <property name="text">
                <string>Réinitialiser</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="getInformations">
           <item>
            <layout class="QVBoxLayout" name="Informations">
             <item>
              <layout class="QHBoxLayout" name="DroneInformations">
               <item>
                <widget class="QCheckBox" name="Info_checkbox">
                 <property name="text">
                  <string>Informations Général</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="Info_horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="Info_label">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="myResource.qrc">:/new/myResource/img/red.png</pixmap>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="HardwareInformations">
               <item>
                <widget class="QCheckBox" name="Hardware_checkBox">
                 <property name="text">
                  <string>Informations Matériel</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="Hardware_horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="Hardware_label">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="myResource.qrc">:/new/myResource/img/red.png</pixmap>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="MissionManagement">
               <item>
                <widget class="QCheckBox" name="Mission_checkBox">
                 <property name="text">
                  <string>Cartographie</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="Mission_horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="Mission_label">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="myResource.qrc">:/new/myResource/img/red.png</pixmap>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="Photos">
               <item>
                <widget class="QCheckBox" name="Photos_checkBox">
                 <property name="text">
                  <string>Photos</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="Photos_horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="Photos_label">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="pixmap">
                  <pixmap resource="myResource.qrc">:/new/myResource/img/red.png</pixmap>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QProgressBar" name="getInfo_progressBar">
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="separator_bottom">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="GetInformationsButton">
             <property name="text">
              <string>Récupérer les informations</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="myResource.qrc"/>
 </resources>
 <connections/>
</ui>
